.github/workflows/release.yml

```diff
@@ -43,7 +43,14 @@ jobs:
         tf-version: ['2.1.0', '2.2.0rc2']
       fail-fast: false
     steps:
-      - if: github.event_name == 'push'
+      - uses: actions/github-script@0.3.0
+        id: author-date
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const commit_details = await github.git.getCommit({owner: context.repo.owner, repo: context.repo.repo, commit_sha: context.sha});
+            return commit_details.data.author.date
+      - if: github.event_name == 'pull_request'
```

Comment:
Using `pull_request` just for debugging. Will need to be reverted to `push` ... just making a note for myself.